common/mixins.py

```diff
@@ -30,7 +30,8 @@ def clean(self):
         if hasattr(self, 'slugify_field') and hasattr(self, 'slug'):
             self.slug = slugify(getattr(self, self.slugify_field))
 
-            if self.__class__.objects.filter(slug=self.slug).exists():
+            # If pk exists, object exists in db and is being edited by user.
```

Comment:
👍 cool
Could we also update our test cases to edit objects that have already been created to ensure that they have been updated?
Is it part of this ticket?